src/Core/AbstractRequest.php

```diff
@@ -8,11 +8,20 @@
      */
     protected $client;
 
+    protected $asJson = false;
+
     /**
      * @param AbstractClient $client
      */
     public function __construct(AbstractClient $client)
     {
         $this->client = $client;
     }
+
+    public function asJson()
```

Comment:
I think it might be better to add a `getRawResponse()` or similar to the AbstractEntity instead, adding `if ($this->asJson == true) {` to every request class is pretty ugly.